tests/trainer/test_trainer.py

```diff
@@ -98,6 +99,52 @@ def test_optimizer_params_on_device(
         target_device = 'cuda' if device == 'gpu' else 'cpu'
         assert all(param.device.type == target_device for param in parameters)
 
+    @pytest.mark.skipif(version.parse(torch.__version__) < version.parse('2.0.0'),
+                        reason='requires PyTorch 2.0 or higher')
+    @pytest.mark.parametrize('compile_config', [(None, False), ({}, True), ({'mode': 'reduce-overhead'}, True)])
+    def test_torch_compile(self, model: ComposerModel, compile_config: Any):
```

Comment:
can u post screenshot showing this was run locally?

Reply:
Sure. 

**Test output:**
```
collected 3 items

tests/trainer/test_trainer.py::TestTrainerInit::test_torch_compile[compile_config0] PASSED                                                                          [ 33%]
tests/trainer/test_trainer.py::TestTrainerInit::test_torch_compile[compile_config1] PASSED                                                                          [ 66%]
tests/trainer/test_trainer.py::TestTrainerInit::test_torch_compile[compile_config2] PASSED                                                                          [100%]

============================================================================ 3 passed in 1.99s ============================================================================
```